lib/queue_classic_matchers/version.rb

```diff
@@ -1,3 +1,3 @@
 module QueueClassicMatchers
-  VERSION = '1.1.0'
+  VERSION = '4.0.0.alpha1'
```

Comment:
I should've really commented on the preceeding PR, but this got me thinking whether we should drop the `alpha` stuff and start bumping patch/minor versions instead. Not a blocker here though, since it applies to all QC repos

Reply:
I feel like it makes sense to bump all three repos to `4.0.0` soonish. We've used the prerelease version of the main gem in production for ages now so it's definitely stable enough. QC+ and this repo we've tweaked fairly recently so it might make sense to wait a bit more—say until our next cooldown? :)